tests/robustness/engine/engine_test.go

```diff
@@ -52,22 +52,23 @@ func TestEngineWritefilesBasicFS(t *testing.T) {
 	os.Setenv(snapmeta.EngineModeEnvKey, snapmeta.EngineModeBasic)
 	os.Setenv(snapmeta.S3BucketNameEnvKey, "")
 
-	th, eng, err := newTestHarness(t, fsDataRepoPath, fsMetadataRepoPath)
+	ctx := context.TODO()
```

Comment:
Can the kopia testing contexts be used here and reused in a test?

Reply:
Is that `testlogging.Context(t)` you're referring to?